### ISSUE TITLE
Output of \languagevariant is empty

### ISSUE THREAD

**Issue by wehro** [reutenauer/polyglossia#530]
The `\languagevariant` command (described in section 12 of the manual) always seems to give empty output instead of the name of the current language variant.
```
\documentclass{article}
\usepackage{polyglossia}
\setmainlanguage[variant=austrian]{german}
\setotherlanguage[variant=british]{english}

\begin{document}
\languagename
\languagevariant
\selectlanguage{english}
\languagename
\languagevariant
\end{document}
```
The output is “german english”, but it should be “german austrian english british”.

**Comment by jspitz** [comment]
This has been broken by 7472bf69ab221

**Comment by jspitz** [comment]
Fixed at 48346acf8953.
